src/app/components/Toolbar/Features/Contacts/index.tsx

```diff
@@ -1,37 +1,53 @@
-import { useState, useContext } from 'react'
+import { useState, useContext, ReactNode } from 'react'
 import { useSelector } from 'react-redux'
 import { useTranslation } from 'react-i18next'
 import { Box } from 'grommet/es6/components/Box'
 import { Button } from 'grommet/es6/components/Button'
 import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
 import { Inbox } from 'grommet-icons/es6/icons/Inbox'
 import { selectContactsList } from 'app/state/contacts/selectors'
+import { selectIsLockableOrCloseable } from 'app/state/selectIsLockableOrCloseable'
 import { ContactAccount } from './ContactAccount'
 import { AddContact } from './AddContact'
 import { layerScrollableAreaHeight } from './layer'
 
-const ContactsListEmptyState = () => {
-  const { t } = useTranslation()
-
-  return (
-    <Box gap="medium" align="center" pad={{ top: 'large' }}>
-      <Inbox size="36px" color="currentColor" />
-      <Box pad="large">{t('toolbar.contacts.emptyList', 'You have no contacts yet.')}</Box>
-    </Box>
-  )
+type ContactsListEmptyStateProps = {
+  children: ReactNode
 }
 
+const ContactsListEmptyState = ({ children }: ContactsListEmptyStateProps) => (
+  <Box gap="medium" align="center" pad={{ top: 'large' }}>
+    <Inbox size="36px" color="currentColor" />
+    <Box pad="large">{children}</Box>
+  </Box>
+)
+
 export const Contacts = () => {
   const { t } = useTranslation()
   const [layerVisibility, setLayerVisibility] = useState(false)
   const contacts = useSelector(selectContactsList)
+  const isLockableOrCloseable = useSelector(selectIsLockableOrCloseable)
   const isMobile = useContext(ResponsiveContext) === 'small'
+  const isAvailable = isLockableOrCloseable === 'lockable'
```

Comment:
I would use the underlaying `selectUnlockedStatus; unlockedStatus === 'unlockedProfile'`
(like persist.createProfile.startPersisting)